scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1575,13 +1585,17 @@ function Reset-SystemPath {
     Write-Verbose 'Removed IoT Edge directories from system PATH'
 }
 
+function Test-VcRuntimePresent {
+    return Test-Path 'C:\Windows\System32\vcruntime140.dll'
```

Comment:
Is this correct when running in 32-bit PS? Won't it redirect to `C:\Windows\SysWoW64` ?

Reply:
I suppose.  that could only occur on Enterprise and i'm not sure why anybody would run 32-bit PS to install a 64 bit product.  but, I guess it's possible.  however, that path test was what was there before I just moved it into a separate named function to make the intent of the path clear.

Reply:
>i'm not sure why anybody would run 32-bit PS to install a 64 bit product. but, I guess it's possible.

Me neither, but apparently there was enough reason that we added support for it intentionally https://github.com/Azure/iotedge/pull/1293 . I can't remember why, though. Oh well.


>however, that path test was what was there before I just moved it into a separate named function to make the intent of the path clear.

Oh yes, I missed that it was already using System32 before. Never mind then.